src/app/modals/no-ig/no-ig.component.pug

```diff
@@ -3,10 +3,12 @@
     // modal content: Is deleted, now what?
     .modal-content
       .modal-header
-        h4.modal-title([innerHtml]="'NO_IG.HEADING' | translate")
+        h4.modal-title(*ngIf="!noAccessIg", [innerHtml]="'NO_IG.HEADING_NO_EXISTS' | translate")
```

Comment:
I would just make this a ternary in the innerHtml
`[innerHtml]="noAccessIg ? 'NO_IG.HEADING_NO_ACCESS' : 'NO_IG.HEADING_NO_EXISTS' | translate"`

Reply:
<img width="1385" alt="Screen Shot 2020-03-12 at 7 22 57 PM" src="https://user-images.githubusercontent.com/3641288/76583624-f5cdd200-6496-11ea-911f-a43ccf320b84.png">
I can't seem to make this work. It just spits out the literal constant name. I tried wrapping the ternary in parentheses as well. :\

Reply:
![image](https://user-images.githubusercontent.com/3641288/76583684-26157080-6497-11ea-95a3-8f5dd1e4e01f.png)

Reply:
Ok, I was able to make it work but it's still ugly 😛

Reply:
Refactored in fa3dc38